python/ray/rllib/agents/sac/models.py

```diff
@@ -0,0 +1,179 @@
+from collections import OrderedDict
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from .squash_bijector import SquashBijector
+
+
+SCALE_DIAG_MIN_MAX = (-20, 2)
+
+
+class GaussianLatentSpacePolicy(object):
```

Comment:
@richardliaw @ericl seems like at least ddpg implements separate action and action probability networks. I prefer to have the policy as a single entity that supports operations like getting (both symbolic and numeric) actions, log probabilities, etc. Do you guys have any opinion about how you want this to be implemented?

Reply:
We have https://github.com/ray-project/ray/blob/master/python/ray/rllib/models/model.py, which currently supports just the action logits and value function outputs.

I agree that having a single network is cleaner and more flexible than creating two separate ones, though it's not clear what the interface would look like. Perhaps extra optional methods like value()?